themes-default/slim/package.json

```diff
@@ -27,12 +27,13 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@vue/test-utils": "1.0.0-beta.21",
+    "@vue/test-utils": "1.0.0-beta.20",
```

Comment:
I don't think that was intentional. Please revert that.
#4720

Reply:
Bumped version, and tested. Everything should still work.